File: baekjoon/shim/Python/단계별로_풀어보기/정렬/n_10989.py
import sys
n=int(sys.stdin.readline())
arr=[0]*10001
for _ in range(n):
    arr[int(sys.stdin.readline())]+=1
for i in range(10001):
    while(arr[i]>0):
        sys.stdout.write(str(i)+"\n")
        arr[i]-=1

File: baekjoon/shim/Java/단계별로_풀어보기/정렬/n_10989.java
package Study_2022.baekjoon.shim.Java.단계별로_풀어보기.정렬;

import java.io.*;

public class n_10989 {
    public static void main(String[] args)throws IOException{
        BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw=new BufferedWriter(new OutputStreamWriter(System.out));
        int n=Integer.parseInt(br.readLine()), arr[]=new int[10001];
        for(int i=0; i<n; i++)
            arr[Integer.parseInt(br.readLine())]++;
        for(int i=0; i<10001; i++){
            while(arr[i]>0){
                bw.write(i+"\n");
                arr[i]--;
            }
        }
        bw.flush();
        bw.close();
    }
}
